### PR TITLE
Filling out OBC guide index pages

### PR DESCRIPTION
**Pull request by CatGarab**

### CODE REVIEW

**Comment by CatGarab** [comment]
[Update OBC guides](https://trello.com/c/73UJDcYQ/885-update-obc-guides)

File: docs/obc-docs/iobc/index.rst [hunk 1]
```diff
@@ -1,12 +1,41 @@
...
+drivers need to be installed to support them.
+
+Debug Connection
+----------------
+
+The iOBC should be shipped with an FTDI cable.
+This cable should be connected to the programming adapter, which should then be connected to the
+iOBC, to create the debug UART connection. User file transfer will take place using this connection.
+
+Additionally, a SAM-ICE JTAG may also need to  be connected in order to successfully connect with
```
Comment (cantino) [inline docs/obc-docs/iobc/index.rst]: whitespace

File: docs/obc-docs/bbb/index.rst [hunk 2]
```diff
@@ -1,10 +1,46 @@
...
+
+Debug Console Connection
+------------------------
+
+As documented in section 7.5 of the :title:`Beaglebone Black System
+Reference Manual`, an FTDI cable can be connected to the serial debug
+connector in order to establish a debug console connection.
+
+This connection will be passed through to a Kubos Vagrant image as
+`/dev/FTDI`.
```
Comment (cantino) [inline docs/obc-docs/bbb/index.rst]: Anything special the user needs to do to make this happen?

Comment (CatGarab) [reply docs/obc-docs/bbb/index.rst]: Nope. Our Vagrant has USB forwarding rules already set up

**Pull request merged**